fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/RelationTransformer.java

```diff
@@ -394,7 +394,8 @@ public LogicalPlan visitTable(TableRelation node, ExpressionMapping context) {
         Map<Column, ColumnRefOperator> columnMetaToColRefMap = columnMetaToColRefMapBuilder.build();
         List<ColumnRefOperator> outputVariables = outputVariablesBuilder.build();
         LogicalScanOperator scanOperator;
-        if (node.getTable().getType().equals(Table.TableType.OLAP)) {
+        if (node.getTable().getType().equals(Table.TableType.OLAP)
```

Comment:
After this PR https://github.com/StarRocks/starrocks/pull/7896 is merged, you can use node.getTable().isNativeTable()

Reply:
ok.